index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="css/style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header class="main-section">
+
+        <section class="toolbar">
+          <nav class="desktop">
+            <div class="all-item"><h1><a href="#">My Logo</a></h1></div>
```

Comment:
- [ ] Kindly remove the logo from the `nav` tag, only the hamburger menu is supposed to have a `nav` tag.

Reply:
- [ ] Kindly check the alignment of your header section elements,make sure they align as in the design given

<img width="185" alt="image" src="https://user-images.githubusercontent.com/61408860/170553515-4be030fc-85f3-4227-b785-77ac57702395.png">